source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -474,8 +474,13 @@ struct delete_impacts_visitor : public apply_impacts_visitor {
     bool func_on_vj(nt::VehicleJourney& vj) {
         vj.adapted_validity_pattern = vj.validity_pattern;
         ++ nb_vj_reassigned;
-        vj.remove_impact(impact);
-        for (auto i: vj.get_impacts()) {
+        //it don't seem to be possible to capture an attribute of the object
```

Comment:
True, you cannot do `[&member]() {}`. Please remove this useless bad english comment ;)